ff-netF48-server-sdk.csproj

```diff
@@ -33,12 +33,31 @@
         <None Remove="tests\**;examples\**" />
     </ItemGroup>
 
+    <!--
+        When updating a package to the latest version, order is important when linking the dependency library to a specific target .NET version. 
```

Comment:
The trouble that I saw was in the final package stage where it bundled the older version of the library. I did not get any compiler errors using the 6.x library and only ran into the issue at runtime when the DI service attempted to pull up the 6.x object and map it to the 8.x instance.

I don't think anyone will hit a compile time error, though I could be wrong.

Reply:
The following appears to work, would you like me to update it to use the following:

```
        <PackageReference Include="Disruptor" Version="6.0.0" Condition="'$(TargetFramework)' == 'net8.0'" />
        <PackageReference Include="Disruptor" Version="4.0.0" />
        <PackageReference Include="murmurhash" Version="1.0.3" />
        <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
        <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="8.0.2" Condition="'$(TargetFramework)' == 'net8.0'" />
        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="7.7.1" Condition="'$(TargetFramework)' == 'net7.0'" />
        <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.36.0" />

```

Reply:
if, as you say, adding 10.0 and having that use 6.36 is v likely to break, better to break at compile time
so add a comment saying "unless proven otherwise, 6.36/ a catch all will not work - please add a specific reference for your FTFM (probably targetting latest)" ?

Reply:
Updated and simplified comments in .csproj file.

Reply:
Thanks @MCLifeLeader for the and @bartelink for the excellent contributions and discussions here.  I will be able to come to this on Monday with a view to testing these changes and merge/release after that.